notes/BACKEND.md

```diff
@@ -1,3 +1,31 @@
 # Back-end Systems
 
-_Notes for this topic currently under development_
\ No newline at end of file
+Simply put, back-end systems cover aspects of development that don't involve any kind of user interface. This includes API's, scripts, libraries etc. that don't have a user interface, but run on the server side and provide a unified view to one or many consumers.
```

Comment:
Found this online. Found it to be very elegant way to explain int. 
>  back end refer to the separation of concerns between the presentation layer (front end), and the data access layer (back end) of a piece of software, or the physical infrastructure or hardware.

https://en.wikipedia.org/wiki/Front_and_back_ends

Reply:
Addressed